app/editor/extensions/HoverPreviews.tsx

```diff
@@ -13,13 +15,17 @@ interface HoverPreviewsOptions {
 export default class HoverPreviews extends Extension {
   state: {
     activeLinkElement: HTMLElement | null;
+    data: Record<string, any> | null;
+    dataLoading: boolean;
   } = observable({
     activeLinkElement: null,
+    data: null,
+    dataLoading: false,
   });
 
   get defaultOptions(): HoverPreviewsOptions {
     return {
-      delay: 500,
+      delay: 600,
```

Comment:
Purposely set opening delay to be greater than closing delay(500ms) to prevent picking of stale dimensions for card positioning 👇 

https://github.com/outline/outline/assets/15694566/7b351b22-6d8a-4bb1-8a25-ceaf0427b915

Although a consistent solution for this would be to move the closing delay logic from the component to the extension(in `mouseout` handler). But we're bound to keep that logic within the component because we have to prevent the preview card from closing if hovered 😬